java/src/jmri/jmrit/ctc/NBHSignalMast.java

```diff
@@ -78,9 +82,14 @@ public boolean isDanger() {
     @Override
     public void setAppearance(int newAppearance) {}
 
+//  "getAspect()" can return "null" if (for instance) the signal has no rules (i.e. no "Discover" done yet,
```

Comment:
Please use Javadoc comments to discuss how a method works, so they can be shared with others relying on, but not directly editing, the code.

Reply:
I've done so to the best of my ability, as this is the first time I'm doing this, please review.
You'll notice I've found two missing "null" return values from existing JMRI code (not
my code).  I found out by empirical testing that these in fact do return "null",
and are not documented as such.
getAppearanceMap().getSpecificAppearance(SignalAppearanceMap.DANGER)
and
SignalMast.getAspect()
As you wanted documentation for others to follow, I felt that I needed to document
that fact.
Should I mention that in my comments as I did?  Will this prompt someone to fix the
existing documentation?
From your response to this comment, I will learn what to do in the future when
I encounter such problems in existing JMRI documentation.
I don't know how important such things are, or whether I should mention the
problems in the comments as I did.

Greg